content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/vmtransform.rb

```diff
@@ -67,4 +53,6 @@ def main
   end
 end
 
-ManageIQ::Automate::Transformation::TransformationHost::Common::VMTransform.new.main
+if  == __FILE__
+  ManageIQ::Automate::Transformation::TransformationHost::Common::VMTransform.new.main
+end
```

Comment:
@fdupont-redhat Please follow the changes from https://github.com/ManageIQ/manageiq-content/pull/390.  You no longer need the if-block here.